gtfs-netex-test/gtfsprofile.py

```diff
@@ -458,7 +458,12 @@ def projectStopPlaceToStop(stop_place: StopPlace, transformer: Transformer = Non
         if transformer:
             latitude, longitude = transformer.transform(stop_place.centroid.location.pos.value[0], stop_place.centroid.location.pos.value[1])
         else:
-            latitude, longitude = stop_place.centroid.location.latitude, stop_place.centroid.location.longitude
+            if stop_place.centroid == None:  #TODO this is a bad fix for a bad data problem. The correct way would be to omit this kind of StopPlace or to feed the coordinates from the SceduledStopPlace via PSA
+                latitude=0
+                longitude=0
+                print(f'Warning: StopPlace without coordinate {stop_place.public_code} - {stop_place.name}.')
+            else:
+                latitude, longitude = stop_place.centroid.location.latitude, stop_place.centroid.location.longitude
```

Comment:
```suggestion
                latitude, longitude = stop_place.centroid.location.latitude, stop_place.centroid.location.longitude
```
With this it is still not checked if centroid.location exists right?

Reply:
true. I can add this as well. Currently the problem is that not even the Centroid exists.   And the SimplePointStructure really allows for a Centroid to not have coordinates :-(  Will update the check.

Reply:
I think this should be addressed in David's new code.